Source/UI/CommonUI/View/Core/SecureDisplayView.swift

```diff
@@ -11,27 +11,20 @@ import UIKit
 /**
 Secure display view preventing leaking of content
  */
-final class SecureDisplayView: UIView {
+public final class SecureDisplayView: UIView {
+
+    /**
+     Will protect the provided view.
+     */
+    public static func secure(_ view: UIView, acceptsInput: Bool = false) -> SecureDisplayView {
+        SecureDisplayView(secure: view, acceptsInput: acceptsInput)
+    }
```

Comment:
The init currently used in frames can be faulty as it will use the same namespace as the other 2 initialisers which will not create the correct object. This should be the desired approach to secure the view. 

And whilst it is not in use in the current repo, the file is copied across from the source repository to make sure we have no discrepancies getting in without clear diffs in code reviews.